.travis.yml

```diff
@@ -1,7 +1,22 @@
 language: go
 
+matrix:
+  include:
+  - name: amd64
+    os: linux
+    env: ARCH=amd64
+  - name: ppc64le
+    os: linux-ppc64le
+    env: ARCH=ppc64le
+
 before_install:
   - sudo apt-get update -qq
+  - sudo apt-get remove docker-ce
+  - sudo apt-get install docker-ce
+  - sudo mkdir -p ~/.docker
+  - sudo echo '{"experimental":"enabled"}' | sudo tee ~/.docker/config.json
```

Comment:
Is it necessary to config both ``experimental:enabled`` and ``experimental:true``?

Reply:
one for the server side `/etc/docker/daemon.json` and one for the client side `~/.docker/config.json`